### ISSUE TITLE
[Bug]: Scrollbar not available in Dashboard [NC 25]

### ISSUE THREAD

**Issue by EmJotGeh** [nextcloud/server#34732]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

No scrollbar available and not able to scroll down in Dashboard - not all dashboardcards can be seen.

### Steps to reproduce

1.only since the update to Nextcloud 25 
2.
3.

### Expected behavior

Scrollbar available - able to scroll down on dashboard

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
- activity: 2.17.0
  - analytics: 4.2.1
  - announcementcenter: 6.4.0
  - approval: 1.0.10
  - audioplayer: 3.3.1
  - auto_groups: 1.4.1
  - bookmarks: 11.0.4
  - bruteforcesettings: 2.5.0
  - calendar: 4.0.1
  - camerarawpreviews: 0.8.0
  - cfg_share_links: 3.0.0
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.0.1
  - contactsinteraction: 1.6.0
  - cospend: 1.5.2
  - customproperties: 2.0.4
  - dashboard: 7.5.0
  - dav: 1.24.0
  - deck: 1.8.0
  - dicomviewer: 1.2.3
  - drop_account: 2.1.0
  - end_to_end_encryption: 1.10.0
  - event_update_notification: 2.0.0
  - external: 5.0.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_accesscontrol: 1.15.0
  - files_automatedtagging: 1.15.0
  - files_downloadactivity: 1.15.0
  - files_downloadlimit: 1.0.1
  - files_external: 1.17.0
  - files_lock: 24.0.1
  - files_pdfviewer: 2.6.0
  - files_retention: 1.14.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - files_zip: 1.1.2
  - fileslibreofficeedit: 1.1.0
  - firstrunwizard: 2.14.0
  - flow_notifications: 1.4.0
  - forms: 3.0.0
  - geoblocker: 0.5.7
  - groupfolders: 13.0.0
  - guests: 2.3.0
  - imageconverter: 1.3.4
  - login_notes: 1.1.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - mail: 2.0.3
  - maps: 0.2.1
  - memories: 4.2.2
  - music: 1.6.0
  - nextcloud_announcements: 1.14.0
  - notes: 4.6.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - ojsxc: 5.0.0
  - password_policy: 1.15.0
  - passwords_handbook: 2022.10.20
  - photos: 2.0.0
  - picker: 1.0.3
  - polls: 4.0.0
  - previewgenerator: 5.1.0
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - qownnotesapi: 22.10.0
  - radio: 1.0.3
  - recommendations: 1.4.0
  - related_resources: 1.0.1
  - richdocuments: 7.0.0
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - sharelisting: 1.1.1
  - sharerenamer: 3.1.0
  - side_menu: 3.0.1
  - spreed: 15.0.0
  - support: 1.8.0
  - suspicious_login: 4.3.0
  - systemtags: 1.15.0
  - tasks: 0.14.5
  - text: 3.6.0
  - theming: 2.0.0
  - theming_customcss: 1.12.0
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_migration: 2.0.1
  - user_status: 1.5.0
  - viewer: 1.9.0
  - w2g2: 3.3.9
  - weather_status: 1.5.0
  - welcome: 1.0.2
  - whiteboard: 0.0.3
  - workflow_ocr: 1.25.0
  - workflow_pdf_converter: 1.10.0
  - workflowengine: 2.7.0
Disabled:
  - admin_audit
  - appointments: 1.12.8
  - apporder: 0.15.0
  - audioplayer_editor: 0.3.0
  - audioplayer_sonos: 1.3.0
  - breezedark: 24.0.2
  - carnet: 0.24.5
  - checksum: 1.1.4
  - cms_pico: 1.0.19
  - collectives: 1.5.0
  - cookbook: 0.9.15
  - data_request: 1.11.0
  - diary: 0.0.8
  - drawio: 1.0.3
  - duplicatefinder: 0.0.15
  - emlviewer: 1.0.2
  - encryption: 2.11.0
  - extract: 1.3.5
  - file_sharing_invalid_notifier: 0.1.1
  - files_antivirus: 3.3.1
  - files_linkeditor: 1.1.11
  - files_markdown: 2.3.6
  - files_mindmap: 0.0.26
  - files_reader: 1.5.3
  - files_readmemd: 1.2.2
  - files_texteditor: 2.14.0
  - files_trackdownloads: 1.11.0
  - flowupload: 1.1.3
  - gestion: 2.1.7
  - gpxedit: 0.0.14
  - gpxmotion: 0.1.0
  - gpxpod: 4.3.0
  - group_default_quota: 0.1.4
  - groupquota: 0.1.8
  - health: 1.5.1
  - impersonate: 1.11.0
  - jitsi: 0.15.0
  - keeweb: 0.6.9
  - limit_login_to_ip: 3.1.0
  - metadata: 0.16.0
  - ncdownloader: 1.0.1
  - news: 18.2.0
  - passwords: 2022.10.20
  - pdf_downloader: 1.0.6
  - permissions_overwrite: 0.1.7
  - phonetrack: 0.7.0
  - podcast: 0.3.1
  - quickaccesssorting: 1.1.4
  - quicknotes: 0.8.1
  - quota_warning: 1.14.0
  - richdocumentscode_arm64: 22.5.502
  - riotchat: 0.13.10
  - rocketchat_nextcloud: 0.9.6
  - sharepermissions: 1.0.1
  - socialsharing_email: 2.5.0
  - socialsharing_telegram: 2.5.0
  - survey_client: 1.10.0
  - tables: 0.2.1
  - talk_matterbridge: 1.25.2
  - talk_simple_poll: 1.3.1
  - timemanager: 0.3.3
  - timetracker: 0.0.75
  - transfer: 0.6.0
  - twofactor_totp
  - user_ldap
  - user_retention: 1.7.0
  - user_usage_report: 1.8.0
  - video_converter: 1.0.5
  - weather: 1.7.6
  - workflow_media_converter: 1.4.0
  - workflow_script: 1.9.0
```

### Additional info

only since the update to Nextcloud 25

**Comment by szaimen** [comment]
hi, can you post a screenshot?

**Comment by EmJotGeh** [comment]
![Screenshot_NC25_Dashboard_edit jpeg](https://user-images.githubusercontent.com/16000642/197305657-d8ea742f-5aa8-4a94-bc18-dad300d57caa.jpg)

**Comment by EmJotGeh** [comment]
The radio app was the reason why scrolling was not possible. The topic can be closed. Thanks :-)

**Comment by szaimen** [comment]
Thanks!

**Comment by q4z1** [comment]
I cannot confirm. Scrolling does not work even without the radio app widget activated.
For me the responsible css is in
`https://github.com/nextcloud/server/blob/master/apps/dashboard/src/DashboardApp.vue` `Line 641`
Removing this position style as a quick fix falls back to `position: fixed` which enables the scrollbar.

**Comment by jamiew0w** [comment]
> I cannot confirm. Scrolling does not work even without the radio app widget activated. For me the responsible css is in `https://github.com/nextcloud/server/blob/master/apps/dashboard/src/DashboardApp.vue` `Line 641` Removing this position style as a quick fix falls back to `position: fixed` which allows the scrollbar.

Does this require building manually? I've tested with modifying the dashboardapp.js but no luck unfortunately. I'm setting up a brand new install and running into this, it's very frustrating.

I'm guessing it can be built from https://github.com/nextcloud/nextcloud-vue and then just copied across.

**Comment by q4z1** [comment]
I'd rather put custom css with an override rule like this until the devs fix the issue:

`body #content {
	position: absolute!important;
}`

[(https://apps.nextcloud.com/apps/theming_customcss)]

**Comment by ldpr** [comment]
> I'd rather put [custom css](https://apps.nextcloud.com/apps/theming_customcss) with an override rule like this until the devs fix the issue:
> 
> `body #content { position: absolute!important; }`

Appreciate it :)
In the end I just nuked it and installed NC24 instead.

**Comment by jamiew0w** [comment]
> I'd rather put [custom css](https://apps.nextcloud.com/apps/theming_customcss) with an override rule like this until the devs fix the issue:
> 
> `body #content { position: absolute!important; }`

Appreciate it.
I just nuked NC25 and installed NC24 instead, no issues :)
Thanks!

**Comment by boredazfcuk** [comment]
> Appreciate it. I just nuked NC25 and installed NC24 instead, no issues :) Thanks!

@jamiew0w Did you ever manage to get upgraded?

I'm seeing exactly the same issue. I get no scroll bars and using the scroll wheel on the mouse doesn't do anything.

I know there is stuff off the bottom of the screen, because if I zoom right out in Firefox, it all comes into view (just too small to read).

I also noticed that some directories were not displaying the correct amount of folders/files... Like it would say "23 folders and 2 files" at the bottom of the file list in the Web UI, but only display 9 folders and 0 files.

I tried to upgrade to v26 to see if that fixed things, but it was the same. I've rolled my container back to v24, restored the website volume and the database backup, so now it works again.

I'm running the 24-fpm container and reverse proxying with nginx... Possibly something to do with that?
